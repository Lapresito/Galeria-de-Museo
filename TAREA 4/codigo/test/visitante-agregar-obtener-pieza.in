
crearVisitante 1 Margarita Mendez 31
obtenerPiezasFavoritasVisitante
agregarPiezasFavoritasVisitante 2 1 2
obtenerPiezasFavoritasVisitante
agregarPiezasFavoritasVisitante 2 1 2
obtenerPiezasFavoritasVisitante
agregarPiezasFavoritasVisitante 5 6 2 1 5 4
obtenerPiezasFavoritasVisitante
liberarVisitante
crearVisitante 2 Violeta Alvarez 54
obtenerPiezasFavoritasVisitante
agregarPiezasFavoritasVisitante 5 10 20 30 40 50
obtenerPiezasFavoritasVisitante
agregarPiezasFavoritasVisitante 5 12 22 32 42 52
obtenerPiezasFavoritasVisitante
agregarPiezasFavoritasVisitante 5 1 2 3 4 5
obtenerPiezasFavoritasVisitante
liberarVisitante
Fin
