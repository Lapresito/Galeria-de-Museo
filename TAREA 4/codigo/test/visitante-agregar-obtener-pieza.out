Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> Visitante creado de forma exitosa.
2> Piezas favoritas del visitante:

3> Se ejecuto agregarPiezasFavoritasTVisitante exitosamente.
4> Piezas favoritas del visitante:
1 2 
5> Se ejecuto agregarPiezasFavoritasTVisitante exitosamente.
6> Piezas favoritas del visitante:
1 2 
7> Se ejecuto agregarPiezasFavoritasTVisitante exitosamente.
8> Piezas favoritas del visitante:
1 2 4 5 6 
9> Visitante liberado con exito.
10> Visitante creado de forma exitosa.
11> Piezas favoritas del visitante:

12> Se ejecuto agregarPiezasFavoritasTVisitante exitosamente.
13> Piezas favoritas del visitante:
10 20 30 40 50 
14> Se ejecuto agregarPiezasFavoritasTVisitante exitosamente.
15> Piezas favoritas del visitante:
10 12 20 22 30 32 40 42 50 52 
16> Se ejecuto agregarPiezasFavoritasTVisitante exitosamente.
17> Piezas favoritas del visitante:
1 2 3 4 5 10 12 20 22 30 32 40 42 50 52 
18> Visitante liberado con exito.
19> Fin.
